lib/workers/repository/process/extract-update.ts

```diff
@@ -19,12 +19,24 @@ export type ExtractResult = {
   packageFiles: Record<string, PackageFile[]>;
 };
 
+interface StatsResult {
+  fileCount: number;
+  depCount: number;
+}
+
+interface Stats {
+  managers: Record<string, StatsResult>;
+  total: StatsResult;
+}
+
 // istanbul ignore next
-function extractStats(packageFiles: Record<string, PackageFile[]>): any {
+function extractStats(
+  packageFiles: Record<string, PackageFile[]>
+): Stats | null {
   if (!packageFiles) {
-    return {};
+    return null;
```

Comment:
do we check the result? before it doesn't return null

Reply:
The only place it's being used is `logger.info` statement